src/prepare-styles/typography.scss

```diff
@@ -45,4 +43,42 @@
     font-size: 32px;
     line-height: 41px;
   }
+}
+
+.card_container-title{
+  font-family: 'Mont-SemiBold', sans-serif;
+  font-size: 14px;
+  color: $grayPrimary;
+  line-height: 21px;
+}
+
+.card_container-prices-salePrice{
+  font-family: 'Mont-Bold', sans-serif;
+  font-weight: 800;
+  font-size: 22px;
+  color: $grayPrimary;
+  line-height: 30.8px;
+}
+
+.card_container-prices-fullPrice{
+  font-family: 'Mont-SemiBold', sans-serif;
+  font-size: 22px;
+  color: $graySecondary;
+  line-height: 28.12px;
+  text-decoration: line-through;
+}
+
+.card_container-about-characteristics-title{
+  font-family: 'Mont-Bold', sans-serif;
+  font-size: 12px;
+  color: $graySecondary;
+  line-height: 15.34px;
+}
+
+.card_container-about-characteristics-value{
+  font-family: 'Mont-Bold', sans-serif;
+  font-size: 12px;
+  color: $grayPrimary;
+  line-height: 15.34px;
+  text-transform: uppercase;
```

Comment:
I believe typography that is specific for card is better to move into Card.scss file